### DISCUSSION TITLE
Trying to run Fief using Docker Compose

### DISCUSSION THREAD

**Discussion by anirudhprabhakaran3** [fief-dev/fief#197] [Q&A] [answered]
Hello! I am trying to use Fief and install it on my system using Docker Compose. I am referring to the [Self Hosting](https://docs.fief.dev/self-hosting/quickstart/) page, and am trying to run the Quickstart step. However, while I am running this, I am getting the following error:

![image](https://github.com/fief-dev/fief/assets/76864239/06df75c6-fb78-4fc9-9438-1a165f7c0a92)

I understand that the quickstart is actually supposed to create these variables for me, so that I can use it in my `.env` file, However, I am not able to do so.

System:  Windows 11 with WSL Ubuntu 22.04.2 LTS

Is the issue something to do with me trying to run it on WSL? Thank you very much!

**Comment by fief-bailiff** [comment]
Hail, @anirudhprabhakaran3 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by frankie567** [comment]
Yes, that's a bug I spotted and fixed in https://github.com/fief-dev/fief/commit/b10045b8ebca8099c7dabd7c1e9e300a906d052b. I'll release it soon!

**Comment by auditt98** [comment]
Hello, is this fix released?

**Comment by frankie567** [comment, marked as answer]
Fix released today in [0.24.4](https://github.com/fief-dev/fief/releases/tag/v0.24.4)

**Comment by anirudhprabhakaran3** [comment]
Thank you very much!
